pkg/network/java/testutil/JavaClientSimulator/src/main/java/Clients/Main.java

```diff
@@ -0,0 +1,104 @@
+package Clients;
+import org.apache.commons.cli.*;
+
+import java.io.IOException;
+
+public class Main {
+
+    public enum ClientType{
+        apache,
+        okhttp,
+        httpclient,
+        urlconnection,
+        unsupported,
+    }
+
+
+    private static void printHelp(Options options){
+        HelpFormatter formatter = new HelpFormatter();
+        formatter.printHelp("java -jar JavaClients client=<client> url=<url>\n",
+                "",
+                options,
+                "\nprovide the url WITHOUT the protocol scheme (always using https)");
```

Comment:
maybe missed that in the code, but where did you set "do not verify ssl cert"?

Reply:
bump

Reply:
added